app/routes.py

```diff
@@ -63,3 +64,37 @@ def post_kml():
         ),
         201
     )
+
+
+@app.route('/kml/<kml_admin_id>', methods=['GET'])
+def get_id(kml_admin_id):
+    logger.debug("entering route /kml/%s", kml_admin_id)
+    enforcer = DynamoDBFilesHandler(
+        table_name=AWS_DB_TABLE_NAME,
+        bucket_name=AWS_S3_BUCKET_NAME,
+        table_region=AWS_DB_REGION_NAME,
+        endpoint_url=AWS_DB_ENDPOINT_URL
+    )
+    item = enforcer.get_item(kml_admin_id)
+
+    logger.debug(item)
```

Comment:
Log should be a bit more verbose.

Reply:
this was a forgotten log from development. I'm removing it entirely